### ISSUE TITLE
`Calling unknown method: craft\\models\\FieldLayout::getFields()`

### ISSUE THREAD

**Issue by jaydensmith** [verbb/formie#942]
Hey @engram-design,

Looks like this instance of `$fieldLayout->getFields()` hasn't been updated to `$fieldLayout->getCustomFields()` so it's throwing an error when using GraphQL.

https://github.com/verbb/formie/blob/072615d5910a334403c2000270a576c1f4610b16/src/gql/types/generators/FormGenerator.php#L89-L96

**Comment by engram-design** [comment]
Missed this one, fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.14](https://github.com/verbb/formie/releases/tag/2.0.0-beta.14)
